.github/workflows/release.yml

```diff
@@ -149,7 +150,7 @@ jobs:
         run: |
           echo "::set-output name=rustc::$(rustc --version)"
       - name: Build Node
-        run: RUSTFLAGS="-C opt-level=3" cargo build --release --verbose --all
```

Comment:
Why was this not working, I don't understand ?

Reply:
I'm not sure. I've tested this many times in my own environment. I'm just poking at things (and keeping the ferrari busy) at this point -- any ideas?

Reply:
For me it was working as expected, why do you say it was not working ?